### PR TITLE
getAuditableEvents() return cofig

### PR DESCRIPTION
**Pull request by vyacheslav-kara**

### CODE REVIEW

**Comment by quetzyg** [comment]
At least add a description to this PR. Didn't you open a ticket about this? Mention this solves it. Also, where's the updated audit configuration?

You basically had to copy and paste my earlier [comment](https://github.com/owen-it/laravel-auditing/pull/340#issuecomment-344338589).

**Comment by vyacheslav-kara** [comment]
I am sorry am new to github this is my first Pull request

**Comment by quetzyg** [comment]
Sure mate, just update your PR and we'll merge this.

**Comment by quetzyg** [comment]
Mate, you don't have to close this PR. Just commit the `audit.php` file changes and push it.

File: config/audit.php [hunk 1]
```diff
@@ -78,4 +78,20 @@
     */
 
     'console' => false,
+
```
Comment (quetzyg) [inline config/audit.php]: It was supposed to go after the `'user'` stuff, but fuck it, I'll fix it later.

File: config/audit.php [hunk 2]
```diff
@@ -92,6 +92,6 @@
         'created',
         'updated',
         'deleted',
-        'restored',
```
Comment (quetzyg) [inline config/audit.php]: You still have a white space, dude.

**Comment by quetzyg** [comment]
Nicely done @vyacheslav-kara. You'll get the hang of it 😉

**Comment by vyacheslav-kara** [comment]
Хahaha thanks that was hard formatting 😄  Now I go to celebrate my first PR 🎆

**Pull request merged**